composer.json

```
@@ -8,6 +8,12 @@
     "require": {
         "consolidation/robo": "^1.0"
     },
+    "require-dev": {
+        "erusev/parsedown": "^1.7",
+        "php-webdriver/webdriver": "^1.12",
+        "symfony/css-selector": "^3.3",
+        "symfony/dom-crawler": "^3.3"
```

```suggestion
        "symfony/dom-crawler": "^4.4"
```
It is lowest supported version